webapp/src/app/components/dialogs/settings/settings.component.ts

```diff
@@ -56,23 +74,33 @@ export class SettingsComponent implements OnInit {
 		this.setIcon(valid);
 		if (valid) {
 			this.folderFormControl.setErrors(null);
+			return true;
 		} else {
 			this.folderFormControl.setErrors({
 				invalid: true
 			});
+			return false;
 		}
 	}
 	
 	save() {
-		this.electron.saveAssetsPath(this.folderFormControl.value);
+		if (Globals.isElectron) {
+			this.electron.saveAssetsPath(this.folderFormControl.value);
+			const ref = this.snackBar.open('Changing the path requires to restart the editor', 'Restart', {
```

Comment:
Only display this if path was actually changed

Reply:
That's a general issue with the map editor.